### DISCUSSION TITLE
When are we going to write a Deliverable  Report?

### DISCUSSION THREAD

**Discussion by Guanweiyi1999** [ZIYANGSONG2003/AI-Enhanced-WordPress-Development-Toolkit#163] [General]
Is that a Design Specification Report? or it just shows what we have done

**Comment by 2Jus2** [comment]
I will go with Google drive and underneath the technical documentation section

**Reply by Guanweiyi1999** [reply]
Great, are there clear requirements for writing this report? I am still a little unclear about the content.

**Reply by Guanweiyi1999** [reply]
Refer to Deliverable A1: Degree Validator - Test plans?
